explainer-server/app/models/ExplainerStore.scala

```diff
@@ -46,16 +48,16 @@ class ExplainerStore @Inject() (config: Config) extends ExplainerAtomImplicits
           }
         }
       }
-      val updatedExplainer = buildAtomWithDefaults(explainer.id, newExplainerAtom, explainer.contentChangeDetails.revision+1)
+      val updatedExplainer = buildAtomWithDefaults(user, explainer.id, newExplainerAtom, explainer.contentChangeDetails.revision+1)
       explainerDB.store(updatedExplainer)
       updatedExplainer
     }
   }
 
-  def create(): Future[Atom] = {
+  def create(user: com.gu.pandomainauth.model.User): Future[Atom] = {
     val uuid = java.util.UUID.randomUUID.toString
     val explainerAtom = ExplainerAtom("-", "-", DisplayType.Expandable)
-    val explainer = buildAtomWithDefaults(uuid, explainerAtom, 1)
+    val explainer = buildAtomWithDefaults(user, uuid, explainerAtom, 1)
     explainerDB.store(explainer)
     Future(explainer)

```

Comment:
Add a publish function here which updates the published field of contentChangeDetails